packages/payment-detection/src/thegraph/__mocks__/client.ts

```diff
@@ -3,6 +3,7 @@ import type { TheGraphClient } from '../client';
 export const theGraphClient: TheGraphClient = {
   GetLastSyncedBlock: jest.fn(),
   GetPaymentsAndEscrowState: jest.fn(),
+  GetPaymentsAndEscrowStateForReceivables: jest.fn(),
```

Comment:
Please add `GetPaymentsAndEscrowStateForReceivables` to the mocks defined at:

- test/erc20/fee-proxy-contract.test.ts:412
- test/erc20/thegraph-info-retriever.test.ts:47